styles/drifter.scss

```diff
@@ -36,30 +38,33 @@
     }
 
     &-text {
-      color: #a7a7a7;
-      font-family: "Montserrat-Light";
-      font-size: 14px;
+      color: $color;
```

Comment:
bad name for variable
I don't know what is it.
is it text color or not, is it primary or not?

for example:
`$primary-text-color`